src/qmqtt_client_p.cpp

```diff
@@ -93,19 +93,7 @@ void QMQTT::ClientPrivate::init(const QString& hostName, const quint16 port, con
 
     _hostName = hostName;
     _port = port;
-    if (ssl)
```

Comment:
I do not like this part of the patch. You just drop the ```if (ssl)``` part and call always ```Network(ssl, ignoreSelfSigned)```, also if it might not have been defined at all (```#ifdef``` in src/qmqtt_network.h).

I would stuck with my code and just change line 99 into ```_network.reset(new Network(ssl, ignoreSelfSigned));```, so we do not break anything.